openfisca_us/parameters/usda/disabled.yaml

```
@@ -0,0 +1,18 @@
+description: Program participation that qualify individuals as disabled under USDA rules
+
+values:
+  2021-01-01:
+    - federal_disability_or_blindess_receipt
```

Reduce indentation and fix typo below as well
```suggestion
  - federal_disability_or_blindness_receipt
```

---

got the indentation, what typo?

---

`blindess` -> `blindness` :)